doc/user/content/sql/functions/extract.md

```diff
@@ -16,17 +16,56 @@ Parameter | Type | Description
 ----------|------|------------
 _val_ | [`date`](../../types/date), [`timestamp`](../../types/timestamp), [`timestamp with time zone`](../../types/timestamptz) | The value from which you want to extract a component.
 
+### Arguments
+
+`EXTRACT` supports multiple synonyms for most time periods.
```

Comment:
suggestion, feel free to ignore:  wdyt of 

> `EXTRACT` supports multiple synonyms for most time periods that can be used interchangeably.

I know it's redundant, but the statement feels a little terse, and for some reason my brain had to think a little bit when i first read it

Reply:
Does 

> `EXTRACT` supports the use of different synonyms to denote most time periods.

work better?

Reply:
👍 thank you!